src/query.js

```diff
@@ -463,6 +469,11 @@ export default class Query {
     ];
   }
 
+  sortColumnExists(columnName) {
+    let expression = this.sorting.expressions.find(sort => sort.field == `${columnName}.name`)
+    return expression ? true : false
+  }
+
```

Comment:
Isn't there any way to get rid of the variable?

!!this.sorting.expressions.find(sort => sort.field == `${columnName}.name`)

Amazing job by the way.